src/modules/SearchPage/components/SearchBar/index.tsx

```diff
@@ -0,0 +1,24 @@
+import * as S from './styled';
+import { SearchIcon } from 'assets/images';
+import React, { ChangeEvent } from 'react';
+
+type Props = {
+  label: string;
+  inputValue: string;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+};
+
+export class SearchBar extends React.Component<Props, unknown> {
+  render() {
+    const { inputValue, label, onChange } = this.props;
+    return (
+      <S.SearchBar title="search-bar">
+        <S.Label title="local-storage-label">{label}</S.Label>
+        <S.InputWrapper>
+          <SearchIcon />
+          <S.Input onChange={onChange} title="local-storage-input" value={inputValue} type="text" />
```

Comment:
what this title prop is?

Reply:
I could see people adding title to have easier access to elements when testing. I realized this is not the best practice and I removed titles